app/lib/forms/ehco_new_headteacher.rb

```diff
@@ -1,36 +1,36 @@
 module Forms
-  class AsoNewHeadteacher < Base
-    VALID_ASO_NEW_HEADTEACHER_OPTIONS = %w[yes no].freeze
+  class EhcoNewHeadteacher < Base
+    VALID_EHCO_NEW_HEADTEACHER_OPTIONS = %w[yes no].freeze
```

Comment:
How about `VALID_EHCO_NEW_HEADTEACHER_OPTIONS = YES_NO` or too much DSL'y for your liking?

Reply:
Yeah i feel that starts to look confusing but maybe im being old fashioned. 😂